Description: Added keyword field in edb.desktop and changed the edb icon.
Author: Marcio de Souza Oliveira <m.desouza20@gmail.com>
Last-Update: 2017-04-30

Index: edb-debugger-0.9.21/edb.desktop
===================================================================
--- edb-debugger-0.9.21.orig/edb.desktop
+++ edb-debugger-0.9.21/edb.desktop
@@ -3,7 +3,8 @@ Name=edb
 GenericName=edb debugger
 Comment=edb debugger 
 Exec=edb
-Icon=edb
+Icon=edb48-logo
 Terminal=false
 Type=Application
 Categories=Development;Qt;Debugger;
+Keywords=debugger;development;graphical
